pkgs/applications/radio/uhd/default.nix

```diff
@@ -93,6 +93,7 @@ stdenv.mkDerivation rec {
   nativeBuildInputs = [
     cmake
     pkg-config
+    python3
   ]
     # If both enableLibuhd_Python_api and enableUtils are off, we don't need
```

Comment:
Thanks for the PR. It'd be better to add `pythonEnv` unconditionally to `nativeBuildInputs` - it'll evaluate to a python environment without any python packages in it.

I remember I was thinking about it when I wrote the comment here, and I haven't checked cross compilation.

Reply:
It'd also be nice to test cross compilation with `enablePythonApi = true;`, and comment whether it works or not.